docs/all_development/contributing/finding_help.rst

```diff
@@ -0,0 +1,84 @@
+.. Information copied from edx-documentation/shared/preface.rst
+
+.. _Finding help for your development work:
+
+#################################################
+Finding help for your development work
+#################################################
+
+*************
+Getting help
+*************
+
+The `Getting Help`_ page in the Open edX Portal lists different
+ways that you can ask, and get answers to, questions.
+
+.. _Getting Help: https://open.edx.org/getting-help
+
+********************
+Wikis and web sites
+********************
+
+The `Open edX Portal`_ is the entry point for new contributors.
+
+The edX Engineering team maintains an `open Confluence wiki`_, which
+provides insights into the plans, projects, and questions that the edX Open
+Source team is working on with the community.
+
+*******************************************
+Resources for the entire Open edX community
+*******************************************
+
+Hosting providers, platform extenders, core contributors, and course staff all
+use Open edX. EdX provides release-specific documentation, as well as the
+latest version of all guides, for Open edX users.
+
+See `Open edX documentation`_ for information in areas other than software development.
+
+
+.. _Building and Running an edX Course: http://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/
```

Comment:
A lot of these links aren't needed on this page.

Reply:
I don't mean to be difficult, but which links aren't needed? I need more clarity in terms of the change to be made.